ui/src/main/js/view/remove-sidepanel.js

```diff
@@ -27,4 +27,5 @@ var jqProxy = require('../jQuery');
 exports.render = function (message, onElement) {
     var $ = jqProxy.getJQuery();
     $('div#side-panel').remove();
+    $('div#main-panel').addClass("fullscreen");
```

Comment:
Absolutely - once that capability is in core, it's a much better way of doing things 👍